src/script/api/script_airport.cpp

```diff
@@ -156,7 +156,7 @@
 	return AirportGetNearestTown(as, AirportTileTableIterator(as->table[0], tile), dist)->index;
 }
 
-/* static */ uint16 ScriptAirport::GetMaintenanceCostFactor(AirportType type)
+/* static */ SQInteger ScriptAirport::GetMaintenanceCostFactor(AirportType type)
```

Comment:
The second line of the commit message should go, since that has already been done in the previous commit.